rollup/da_syncer/da_queue.go

```diff
@@ -4,25 +4,23 @@ import (
 	"context"
 	"errors"
 
-	"github.com/scroll-tech/go-ethereum/log"
+	"github.com/scroll-tech/go-ethereum/core/rawdb"
 	"github.com/scroll-tech/go-ethereum/rollup/da_syncer/da"
 	"github.com/scroll-tech/go-ethereum/rollup/da_syncer/serrors"
 )
 
 // DAQueue is a pipeline stage that reads DA entries from a DataSource and provides them to the next stage.
 type DAQueue struct {
-	l1height     uint64
-	initialBatch uint64
+	l1height uint64
 
 	dataSourceFactory *DataSourceFactory
 	dataSource        DataSource
 	da                da.Entries
 }
 
-func NewDAQueue(l1height uint64, initialBatch uint64, dataSourceFactory *DataSourceFactory) *DAQueue {
```

Comment:
Recovery doesn't use `initialBatch` anymore?

Reply:
this is now handled via https://github.com/scroll-tech/go-ethereum/blob/ee53754057232453e9c2804ad68595732317a999/rollup/da_syncer/batch_queue.go#L147